data/Dels/Dels missions.txt

```diff
@@ -698,6 +698,163 @@ mission "First Contact: Dels Ufulre"
+			`As your ship sinks through the atmosphere of <planet>, your communicator buzzes. The color is orange: answer required.`
+			label answer
+			`	The it reads, "Officer <first>, you are required to go to <destination> by <date>. There you will recieve further orders."`
+			choice
+				`(Accept)`
+					accept
+				`"I don't care what consequences you give me, I'm not going!"`
+					goto decline
+			label decline
+				action
+					"reputation: Dels Kaote" -= 1
+				fine 10000
+			`Officer <last>, you have dissapinted us. The Kaote Navy will not tolerate such a responce, and as such you will be fined 10,000 credits.`
+	on fail
+		"reputation: Dels Kaote" -= 1
+		fine 5000
+		dialog `Officer <last>, we did not expect you to be so lazy! The Kaote Navy cannot tolerate that, and as such you will be fined 5,000 credits.`
+	on abort
+		"reputation: Dels Kaote" -= 3
+		fine 20000
+		dialog `Officer <first>, the Kaote Navy will not tolerate liars! You will be fined 20,000 credits for your disloyal behavior!`
+	on complete
+		"dels officer: report mission available" = 1
+		"Dels Officer: report complete" += 1
+		#Cap the factor to 25 to avoid mission not offering again if done for 30 times.
+		#So minimal chance of offering is 5%
+		"Dels Officer: report complete" <?= 25
+
+#mission "Dels Officer: Participate in defense"
+#	landing
+#	repeat
+#	priority
+#	name `Drive off the Ufulre`
+#	source Delro
+#	destination
+#		government "Dels Kaote" Dels
+#		near Alkurud 0 2
+#	description `There is an Ufulre attack on <destination>. You must participate in driving it off.`
+#	to offer
+#		has "dels officer: report mission available"
+#	on offer
+#		clear "dels officer: report mission available"
+#		conversation
+#			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <last>, you are requested for defense at <destination>. There is an Ufulre attack there."`
+#			`	You prepare to launch.`
+#				accept
+#	on abort
+#		"reputation: Dels Kaote" -= 1
+#		fine 10000
+#		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
+#
+#	npc evade
+#		government "Dels Ufulre"
+#		personality
+#			confusion 30
+#			plunders harvests target # no staying so they can flee
+#		fleet "Dels Ufulre Raid" 2
+#		system destination
+#
+#	npc
+#		government "Dels Kaote"
+#		personaility
+#			confusion 10
+#			heroic opportunistic frugal staying
+#		fleet "Dels Kaote"
+#		system destination
+#	on complete
+#		payment 100000
+#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`
+
+mission "Dels Officer: Patrol Systems"
+	landing
+	repeat
+	priority
+	name `Partrol systems`
+	source Delro
+	stopover
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	stopover
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	stopover
+		government "Dels Kaote" Dels
+		near Alkurud 0 2

```

Comment:
Waypoint would probably be better. Stopover means you have to land on the planet which might get tedious after a while.

Reply:
The reason I did stopovers was so I could use the planet filter specifications, and I don't know if they work with waypoints.

Reply:
Never mind, waypoints work with the specifications.